mmv1/products/apphub/Boundary.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'Boundary'
+description: 'Application management boundary.'
+references:
+  guides:
+    'AppHub': 'https://docs.cloud.google.com/app-hub/docs/'
+  api: 'https://docs.cloud.google.com/app-hub/docs/reference/rest/v1/Boundary'
+
+docs:
+self_link: 'projects/{{project}}/locations/{{location}}/boundary'
+# Singleton resource, point create_url to update_url.
+create_url: 'projects/{{project}}/locations/{{location}}/boundary'
+create_verb: 'PATCH'
+update_url: 'projects/{{project}}/locations/{{location}}/boundary'
+update_verb: 'PATCH'
+exclude_delete: true
+id_format: 'projects/{{project}}/locations/{{location}}/boundary'
+import_format:
+ - "projects/{{project}}/locations/{{location}}/boundary"
+
+parameters:
+ - name: 'location'
+   type: String
+   description: 'Optional. The location for the Boundary resource. Must be global.'
```

Comment:
Is this field optional or required? If it is required, `required: true` should be added.